server/core/src/https/oauth2.rs

```diff
@@ -538,6 +538,49 @@ pub async fn oauth2_openid_discovery_get(
     }
 }
 
+#[derive(Deserialize)]
+pub struct Oauth2OpenIdWebfingerQuery {
+    resource: String,
+}
+
+pub async fn oauth2_openid_webfinger_get(
+    State(state): State<ServerState>,
+    Path(client_id): Path<String>,
+    Query(query): Query<Oauth2OpenIdWebfingerQuery>,
+
+    // For the moment this is implemented ignoring the rel's
+    // Query(rel): Query<Vec<String>>,
```

Comment:
```suggestion
```

Reply:
Forgor I had that. Needs to be deleted one way or another. <3